lib/clouds/kub_cloud_ops.py

```diff
@@ -37,6 +37,16 @@
 from lib.remote.network_functions import hostname2ip, check_url, NetworkException
 from shared_functions import CldOpsException, CommonCloudFunctions 
 
+# `extensions/v1beta1` is not allowed after k8s 1.16+
+k8s_version = "apps/v1"
+pykube.objects.DaemonSet.version = k8s_version
+pykube.objects.Deployment.version = k8s_version
+pykube.objects.ReplicaSet.version = k8s_version
+pykube.objects.StatefulSet.version = k8s_version
+pykube.objects.PodSecurityPolicy.version = k8s_version
+pykube.objects.Ingress.version = k8s_version
+pykube.objects.ThirdPartyResource.version = k8s_version
```

Comment:
Third party resources aren't in `apps/v1`. They were deprecated back in Kubernetes 1.7 in favor of CustomResourceDefinitions and fully removed in 1.8.